SECURITY.md

```diff
@@ -0,0 +1,33 @@
+## Fyra's Security Policy
+
+Fyra takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organization.
+
+If you believe you have found a security vulnerability in any Fyra-owned repository that meets [Microsoft's definition of a security vulnerability](https://aka.ms/security.md/definition), please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them via email to [secure@fyra.so](mailto:secure@fyra.so).
```

Comment:
Maybe we want to add this to the contacts section inside the issue chooser to prevent this issue right at its root (see [GitHub Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)).

Reply:
You can take a look at the API client repository's [config.yml](https://github.com/fyra-technologies/api-client/blob/main/.github/ISSUE_TEMPLATE/config.yml) file, it will look like this in the end:

<img width="1185" alt="image" src="https://github.com/fyra-technologies/.github/assets/4563751/66ed0eb4-15db-4459-823a-43ea26316a65">